core/services/fluxmonitor/flux_monitor.go

```diff
@@ -893,6 +893,10 @@ func (p *PollingDeviationChecker) respondToNewRoundLog(log flux_aggregator_wrapp
 		return
 	}
 
+	if !p.isValidSubmission(logger.Default.SugaredLogger, polledAnswer) {
+		return
+	}
+
```

Comment:
Why is this new check necessary?

Reply:
we also want to do this check responding to new round not started by us